smartseq2_single_sample/tasks/hisat2.wdl

```diff
@@ -24,8 +26,49 @@ task HISAT2PE {
     cpu: "4"
   }
   output {
-    File logfile ="${output_name}.log"
-    File metfile ="${output_name}.hisat2.met.txt"
+    File logfile = "${output_name}.log"
+    File metfile = "${output_name}.hisat2.met.txt"
+    File output_bam = "${output_name}.bam"
+  }
+}
+task HISAT2rsem {
+  File hisat2_ref
+  File fq1
+  File fq2
+  String ref_name
+  String output_name
+  String samplename
+  command {
+    tar -zxvf "${hisat2_ref}" 
+    hisat2 -t \
```

Comment:
That's a ton of parameters! Could you link to some documentation that explains why these are suggested as defaults (or if you've changed from defaults, why you've picked these values?)

Reply:
long story.....you can not find them from hisat2 manual. this set of parameters will allow hisat2 work with rsem.  I will add comments for each